risczero/methods/guest/Cargo.toml

```diff
@@ -5,12 +5,15 @@ edition = "2021"
 
 [workspace]
 
-[dependencies.risc0-zkp]
-version = "1.0.0-rc.2"
+[build-dependencies.risc0-build]
+tag = "v0.13.0"
 git = "https://github.com/risc0/risc0.git"
-default-features = false
+
+[profile.release]
+lto = true
+opt-level = 3
```

Comment:
This isn't necessary since `risc0-build` already ensures an optimized build of guest code.